patches/blockprotocol+0.0.1.patch

```diff
@@ -20,48 +48,156 @@ index 205630a..1cf433a 100644
  };
  
  export type BlockProtocolAggregateEntitiesPayload = {
--  entityTypeId?: string;
++  accountId: string;
++  entityId?: string;
+   entityTypeId?: string;
 -  entityTypeVersionId?: string | null;
++  entityTypeVersionId?: string;
    operation: BlockProtocolAggregateOperationInput;
-   accountId?: string;
+-  accountId?: string;
```

Comment:
`accountId` should remain optional - the rest is reordering and can be made in the package rather than here